UniversityofHelsinki/ResearchDataServiceChannel
# Wizard config changes

To test:
drush cr

Verify that:
- /wizard page has now local tasks (View, Settings and edit questions)
- Wizard has a settings page. /admin/config/wizard
- Wizard settings page has config variables for title, content and consultation link url + title
- Verify that these values actually do something in the wizard page (consultation link is shown only on sub pages.)
- Verify that when no terms are loaded a text is displayed ('no content found' or something similar)
- Test out that wizard works as expected.

Still todo:
- Add variable for "no content found" text
- Add localization for config content
